package.json

```diff
@@ -8,18 +8,23 @@
     "build-upload": "PORT=8080 razzle build --noninteractive && tar -zcvf build.tar.gz build && scp build.tar.gz do2:~/www/gmx-stats",
     "test": "razzle test --env=jsdom",
     "start": "PORT=3113 razzle start",
+    "start-win": "set PORT=3113&razzle start",
     "start:prod": "node build/server.js",
     "lint": "eslint $(git ls-files -m | grep -E \".js$\")",
     "pre-commit": "npm run lint"
   },
   "dependencies": {
     "@apollo/client": "^3.4.10",
     "chalk": "^4.1.1",
+    "classnames": "^2.3.1",
     "console-log-level": "^1.4.1",
     "cors": "^2.8.5",
     "cross-fetch": "^3.1.4",
     "ethers": "^5.4.2",
     "express": "^4.17.1",
+    "forever": "^4.0.1",
+    "framer-motion": "^4.1",
+    "got": "^11.8.2",
```

Comment:
i guess `got` and `forever` are not used anymore (at least i removed them in `master` branch). if you don't need the let's remove these deps

Reply:
Removed got and forever.